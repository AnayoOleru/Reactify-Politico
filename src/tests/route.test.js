import React from 'react';
import { shallow } from 'enzyme';
import Route from '../routes/Routes';



describe('Route Component', () => {
    it('should render the route component correctly', () => {
        const component = shallow(<Route />);
        expect(component).toMatchSnapshot();
    });
});
