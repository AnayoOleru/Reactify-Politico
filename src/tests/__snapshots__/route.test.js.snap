// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Route Component should render the route component correctly 1`] = `ShallowWrapper {}`;
